yarn-project/boxes/private-token/src/contracts/src/interface.nr

```diff
@@ -0,0 +1,67 @@
+/* Autogenerated file, do not edit! */
```

Comment:
these are copied directly from `noir-contracts`, to "pin" against a known working version